tests/integration/test_merge_tree_s3/s3_mocks/unstable_proxy.py

```diff
@@ -70,7 +70,6 @@ def do_HEAD(self):
         if random.random() < 0.25 and len(r.content) > 1024 * 1024:
             r.content = r.content[: len(r.content) // 2]
         self.wfile.write(r.content)
-        self.wfile.close()
```

Comment:
This change is not related to the added logging - it's only to remove "ValueError: I/O operation on closed file" in "python/http/server.py" from logs when running tests.